services/ui-src/src/forms/addEditMcparReport/addEditMcparReport.ts

```diff
@@ -0,0 +1,105 @@
+import { ProgramList } from "../../constants";
+import { DropdownOptions } from "types";
+
+// create dropdown options
+const dropdownOptions: DropdownOptions[] = Object.keys(ProgramList).map(
+  (value) => {
+    return {
+      label: ProgramList[value as keyof typeof ProgramList],
+      value,
+    };
+  }
+);
+
+export default {
+  id: "aep",
+  options: {
+    mode: "onChange",
+  },
+  heading: {
+    add: "Add / Copy a MCPAR",
+    subheading:
+      "Complete the form below to generate a MCPAR for your managed care program. You may choose to copy an existing MCPAR report, which will retain many key responses (e.g., plan names, quality measures, etc.). Copied MCPARs will still have several blank fields and all responses can be edited. To create a blank MCPAR form, leave the “copy a report” field as “Select an option”.",
+    edit: "Edit Program",
+  },
+  fields: [
+    {
+      id: "programName",
+      type: "dropdown",
+      validation: "dropdown",
+      props: {
+        label: "Program name (for new MCPAR)",
+        hint: 'Select from an existing program name. If you need to add a new program or rename a program, select "Other, specify".',
+        options: dropdownOptions,
```

Comment:
Curious why you decided to convert this to a component instead of appending "other, specify" to the bottom of a duplicate "copyEligibleReports" dropdown? Or at least following that pattern to avoid having to convert this.

Reply:
On this one, I don't have a lot of context of how we're pulling in the `copyEligibleReports` but that's a great point. We can potentially keep the list of programs in the database and pull them in similarly!